tests/components/zwave_js/conftest.py

```diff
@@ -240,19 +240,19 @@ def create_backup_fixture():
 # State fixtures
 
 
-@pytest.fixture(name="controller_state", scope="session")
+@pytest.fixture(name="controller_state")
```

Comment:
Why are we removing the scope? This will mean the fixture will be read from disk for each test and probably be slower.